### DISCUSSION TITLE
"wiki-" disappears in the beginning of links to wiki headings

### DISCUSSION THREAD

**Discussion by myurkin** [community/community#38719] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

All headings in wikis are converted to links, which are accessed by `.../page#heading`, and multi-word are converted to something like `one-two`. I've noticed that GitHub automatically removes `wiki-` in the beginning of heading links (immediately after `#`), so `.../page#wiki-two` gets replaced by `.../page#two`. This causes problems for headings like "Wiki pages", see for example - https://github.com/adda-team/adda/wiki/UsingEclipse#wiki-pages . The link fails to reach the specified header, but one can scroll down and check that the anchor at the heading shows exactly the same (not operational) link.

This is either a bug or some strange undocumented behavior.

**Comment by ossanna16** [comment]
Hi there @myurkin and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
